db_test.go

```diff
@@ -1,5 +1,4 @@
-// Copyright 2019 The nutsdb Author. All rights reserved.
-//
+// Copyright 2019 The nutsdb Author. All rights reserved.  //
```

Comment:
pls remove this line change in license, it shouldn't be included in this change.